CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## 0.0.1-alpha.7
+
+### Bug fixes
+
+- Implement Python's floor division semantics for `Literal` `int`s ([#18249](https://github.com/astral-sh/ruff/pull/18249))
+- Don't warn `yield` not in function when `yield` is in function ([#18008](https://github.com/astral-sh/ruff/pull/18008))
+- Fix attribute writes to unions/intersections including modules ([#18313](https://github.com/astral-sh/ruff/pull/18313))
+- Fix binary intersection comparison inference logic ([#18266](https://github.com/astral-sh/ruff/pull/18266))
+- Fix instance vs callable subtyping/assignability ([#18260](https://github.com/astral-sh/ruff/pull/18260))
+- Ignore `ClassVar` declarations when resolving instance members ([#18241](https://github.com/astral-sh/ruff/pull/18241))
+- Infer types for `ty_extensions.Intersection[A, B]` tuple expressions ([#18321](https://github.com/astral-sh/ruff/pull/18321))
```

Comment:
(Rephrase it to make it slightly more tailored to the change a user is likely to be interested in)

```suggestion
- Fix crash when hovering over a `ty_extensions.Intersection[A, B]` expression in an IDE context ([#18321](https://github.com/astral-sh/ruff/pull/18321))
```